pizone/zone.py

```diff
@@ -107,15 +107,14 @@ async def set_airflow_min(self, value: int) -> None:
         if self._zone_data['MinAir'] == value:
             return
 
-        async with self._controller._sending_lock:
-            await self._send_command('AirMinCommand', value)
+        await self._send_command('AirMinCommand', value)
 
-            # need to refresh immediatley after updating
-            try:
-                await self._controller._refresh_zone_group(
-                    self._index - self._index % 4)
-            except ConnectionError:
-                pass
+        # need to refresh immediatley after updating
```

Comment:
I think these might need to be inside the sending lock to avoid a race condition.